.github/workflows/icu_merge_ci.yml

```diff
@@ -0,0 +1,33 @@
+name: GHA ICU Merge CI
```

Comment:
nit: consider adding copyright header

```   
# Copyright (C) 2016 and later: Unicode, Inc. and others.
# License & terms of use: http://www.unicode.org/copyright.html
```

Also maybe a comment about  the file,
`# GitHub Action configuration script for ICU post-merge tasks.`

Reply:
Yes, thank you! Norbert, please add those.

Except: The copyright line should read "2021 and later", since this file is new this year.